src/logic/editor_game_base.cc

```diff
@@ -187,44 +183,22 @@ void EditorGameBase::think() {
 	// by a given number of milliseconds
 }
 
-const World& EditorGameBase::world() const {
+const Descriptions& EditorGameBase::descriptions() const {
 	// Const casts are evil, but this is essentially lazy evaluation and the
 	// caller should really not modify this.
-	return *const_cast<EditorGameBase*>(this)->mutable_world();
+	return *const_cast<EditorGameBase*>(this)->mutable_descriptions();
 }
 
-World* EditorGameBase::mutable_world() {
-	if (!world_) {
-		// Lazy initialization of World. We need to create the pointer to the
-		// world immediately though, because the lua scripts need to have access
-		// to world through this method already.
-		ScopedTimer timer("Registering the world took %ums");
-		Notifications::publish(UI::NoteLoadingMessage(_("Loading world…")));
-		world_.reset(new World(description_manager_.get()));
+Descriptions* EditorGameBase::mutable_descriptions() {
+	if (!descriptions_) {
+		// Lazy initialization of Descriptions. We need to create the pointer to the
+		// descriptions immediately though, because the lua scripts need to have access
+		// to descriptions through this method already.
+		ScopedTimer timer("Registering the descriptions took %ums");
+		Notifications::publish(UI::NoteLoadingMessage(_("Loading world and tribes…")));
+		descriptions_.reset(new Descriptions(lua_.get()));
```

Comment:
add some assert that `lua_` is valid

Reply:
I have added a simple assert. It's fulfilled by the constructor

`lua_(lua_interface ? lua_interface : new LuaEditorInterface(this)),`